### PR TITLE
Resumed PositionController for ant_pan and tilt

### PR DESCRIPTION
**Pull request by damianacatanoso**
Ticket: https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-559

There are two antenna joints: pan and tilt. Both of them don't need planning and are moved independently. So there's no need to switch to the JointTrajectoryController. In this branch I resumed the JointPositionController only for the antenna. 

Test:
- launch europa_terminator_workspace.launch
- in rqt>Control and Viz>Message Publisher click to expand the /ant_pan_position_controller/command topic. Tick the box. Change rate to 100. In the 'expression' field write float numbers such as: 0.5, 0.8, -0.5, one at a time. Observe antenna reaching the goal positions in Gazebo.
- make sure your ow_autonomy is in the OW-518 branch.
- run ReferenceMission1 : roslaunch ow_autonomy autonomy_node.launch plan:="ReferenceMission1.plx"
- observe antenna motion (you'll get an error when Unstow is called, that's normal). 
- Let me know if the antenna moves as expected please

### CODE REVIEW

**Review by Samahu — CHANGES_REQUESTED** [review]
You need to revise and brake the the other definition of **antenna_controller** on line 28 into two controllers as you did to the definition on line 58.

**Comment by damianacatanoso** [comment]
> You need to revise and brake the the other definition of **antenna_controller** on line 28 into two controllers as you did to the definition on line 58.

Can you point it out in the code?

**Comment by Samahu** [comment]
> > You need to revise and brake the the other definition of **antenna_controller** on line 28 into two controllers as you did to the definition on line 58.
> 
> Can you point it out in the code?

GitHub PR system doesn't allow me to reference lines of code away from the code changes. I was referring to line 28 of the file `ow_lander/config/ros_controllers.yaml` (sorry I didn't specify this initially).

**Comment by damianacatanoso** [comment]
> > > You need to revise and brake the the other definition of **antenna_controller** on line 28 into two controllers as you did to the definition on line 58.
> > 
> > 
> > Can you point it out in the code?
> 
> GitHub PR system doesn't allow me to reference lines of code away from the code changes. I was referring to line 28 of the file `ow_lander/config/ros_controllers.yaml` (sorry I didn't specify this initially).

Addressed, thanks!

**Review by mogumbo — APPROVED** [review]

**Comment by Samahu** [comment]
@damianacatanoso It seems that the _follow_joint_trajectory_ should have a matching _JointTrajectoryController_ for things to play nicely. Since we are not going to use a JointTrajectoryController for the **antenna tilt and pan joints** - at least for the current release - we need to remove the definition of both `ant_pan_position_controller/follow_joint_trajectory` and `ant_tilt_position_controller/follow_joint_trajectory`.

**Comment by damianacatanoso** [comment]
> @damianacatanoso It seems that the _follow_joint_trajectory_ should have a matching _JointTrajectoryController_ for things to play nicely. Since we are not going to use a JointTrajectoryController for the **antenna tilt and pan joints** - at least for the current release - we need to remove the definition of both `ant_pan_position_controller/follow_joint_trajectory` and `ant_tilt_position_controller/follow_joint_trajectory`.

Addressed.

File: ow_lander/config/ros_controllers.yaml [hunk 1]
```diff
@@ -211,4 +178,15 @@ grinder_controller:
       p: *kp_default
       d: *kd_default
       i: *ki_default
-      i_clamp: *ki_clamp_default
\ No newline at end of file
+      i_clamp: *ki_clamp_default
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 300
```
Comment (Samahu) [inline ow_lander/config/ros_controllers.yaml:184]: Did you change the publish_rate intentionally? what is the reason for this change?

Comment (damianacatanoso) [reply ow_lander/config/ros_controllers.yaml:184]: Oh I put it equal to the control loop frequency, but maybe they don't have to be equal. What's the reason why we choose 50?

Comment (Samahu) [reply ow_lander/config/ros_controllers.yaml:184]: That's the default value set by **moveit setup_assistant**

Comment (Samahu) [reply ow_lander/config/ros_controllers.yaml:184]: So I am not very in favor of changing as this may have other side effects.

Comment (Samahu) [reply ow_lander/config/ros_controllers.yaml:184]: I also recommend keeping the block position in the same place as before. The reason if we every run setup assistant again then it wouldn't change as something that has change.

Comment (damianacatanoso) [reply ow_lander/config/ros_controllers.yaml:184]: ok

**Review by Samahu — APPROVED** [review]
Thanks for working this out!
